boards.txt

```diff
@@ -22771,6 +22771,158 @@ wt32-eth01.menu.EraseFlash.all.upload.erase_cmd=-e
+wt32-sc01-plus.build.flash_freq=80m
+wt32-sc01-plus.build.flash_mode=dio
+wt32-sc01-plus.build.boot=qio
+wt32-sc01-plus.build.partitions=default
+wt32-sc01-plus.build.defines=
+wt32-sc01-plus.build.loop_core=
+wt32-sc01-plus.build.event_core=
+wt32-sc01-plus.build.flash_type=qio
+wt32-sc01-plus.build.psram_type=qspi
+wt32-sc01-plus.build.memory_type=qio_qspi
+
+wt32-sc01-plus.menu.LoopCore.1=Core 1
+wt32-sc01-plus.menu.LoopCore.1.build.loop_core=-DARDUINO_RUNNING_CORE=1
+wt32-sc01-plus.menu.LoopCore.0=Core 0
+wt32-sc01-plus.menu.LoopCore.0.build.loop_core=-DARDUINO_RUNNING_CORE=0
+
+wt32-sc01-plus.menu.EventsCore.1=Core 1
+wt32-sc01-plus.menu.EventsCore.1.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+wt32-sc01-plus.menu.EventsCore.0=Core 0
+wt32-sc01-plus.menu.EventsCore.0.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=0
+
+wt32-sc01-plus.menu.USBMode.hwcdc=Hardware CDC and JTAG
+wt32-sc01-plus.menu.USBMode.hwcdc.build.usb_mode=1
+wt32-sc01-plus.menu.USBMode.default=USB-OTG (TinyUSB)
+wt32-sc01-plus.menu.USBMode.default.build.usb_mode=0
+
+wt32-sc01-plus.menu.CDCOnBoot.cdc=Enabled
+wt32-sc01-plus.menu.CDCOnBoot.cdc.build.cdc_on_boot=1
+wt32-sc01-plus.menu.CDCOnBoot.default=Disabled
+wt32-sc01-plus.menu.CDCOnBoot.default.build.cdc_on_boot=0
+
+wt32-sc01-plus.menu.MSCOnBoot.default=Disabled
+wt32-sc01-plus.menu.MSCOnBoot.default.build.msc_on_boot=0
+wt32-sc01-plus.menu.MSCOnBoot.msc=Enabled (Requires USB-OTG Mode)
+wt32-sc01-plus.menu.MSCOnBoot.msc.build.msc_on_boot=1
+
+wt32-sc01-plus.menu.DFUOnBoot.default=Disabled
+wt32-sc01-plus.menu.DFUOnBoot.default.build.dfu_on_boot=0
+wt32-sc01-plus.menu.DFUOnBoot.dfu=Enabled (Requires USB-OTG Mode)
+wt32-sc01-plus.menu.DFUOnBoot.dfu.build.dfu_on_boot=1
+
+wt32-sc01-plus.menu.UploadMode.cdc.upload.wait_for_upload_port=true
+wt32-sc01-plus.menu.UploadMode.default=UART0 / Hardware CDC
+wt32-sc01-plus.menu.UploadMode.cdc=USB-OTG CDC (TinyUSB)
+wt32-sc01-plus.menu.UploadMode.cdc.upload.use_1200bps_touch=true
+wt32-sc01-plus.menu.UploadMode.default.upload.use_1200bps_touch=false
+wt32-sc01-plus.menu.UploadMode.default.upload.wait_for_upload_port=false
+
+wt32-sc01-plus.menu.PSRAM.enabled=Enabled
+wt32-sc01-plus.menu.PSRAM.enabled.build.defines=-DBOARD_HAS_PSRAM
+wt32-sc01-plus.menu.PSRAM.disabled=Disabled
+wt32-sc01-plus.menu.PSRAM.disabled.build.defines=
+
+wt32-sc01-plus.menu.PartitionScheme.default_8MB=Default (3MB APP/1.5MB SPIFFS)
+wt32-sc01-plus.menu.PartitionScheme.default_8MB.build.partitions=default_8MB
+wt32-sc01-plus.menu.PartitionScheme.default_8MB.upload.maximum_size=3342336
+wt32-sc01-plus.menu.PartitionScheme.tinyuf2=TinyUF2 Compatibility (2MB APP/3.7MB FFAT)
+wt32-sc01-plus.menu.PartitionScheme.tinyuf2.build.custom_bootloader=bootloader_tinyuf2
+wt32-sc01-plus.menu.PartitionScheme.tinyuf2.build.custom_partitions=partitions_tinyuf2
+wt32-sc01-plus.menu.PartitionScheme.tinyuf2.upload.extra_flags=0x410000 "{runtime.platform.path}/variants/{build.variant}/tinyuf2.bin"
```

Comment:
You are missing the `tinyuf2.bin` file in the variant folder.

Reply:
> You are missing the `tinyuf2.bin` file in the variant folder.

Sorry I updated it. Also I am not sure how to squash commits, sorry about that.